AppConnectSwift/CaptureImageViewController.swift

```diff
@@ -31,6 +39,7 @@ class CaptureImageViewController: UIViewController, UIImagePickerControllerDeleg
         self.dismissViewControllerAnimated(true, completion: nil)
         self.image = info[UIImagePickerControllerOriginalImage] as! UIImage
         imageView.image = self.image
+        self.data = self.compressFile() as? NSData
```

Comment:
@kbohlmann-mdsol 
We have a cap of < 10Mb and we might cross that limit with some images. It actually scales down keeping the aspect ratio proper for the image to be saved.
Let me know we want it to be name to scaleDownImage.